spec/controllers/api/v3/analytics/user_analytics_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe Api::V2::Analytics::UserAnalyticsController, type: :controller do
+RSpec.describe Api::V3::Analytics::UserAnalyticsController, type: :controller do
```

Comment:
@kitallis I'm guessing this was an accident. Let me know if this was done intentionally. This change was introduced in the HTN only dashboards PR.

Reply:
The state prior to that was incorrect at some point, the `v3` spec was calling into `v2`. This is the correct state.